dotnet-estimation/dotnet-estimation/Templates/WebAPI/Devon4Net.Application.WebAPI.Implementation/Business/SessionManagement/Service/SessionService.cs

```diff
@@ -74,5 +74,35 @@ public async Task<Session> GetSession(long id)
 
             return (sessionIsValid, null);
         }
+
+        public async Task<Estimation> AddNewEstimation(long sessionId, string VoteBy, int Complexity)
+        {
+           var result = await GetStatus(sessionId);
```

Comment:
Just a small change that could improve readability: You could change the name of the variable `result` because it's used throughout the function but it's not immediately clear what it means without checking out the GetStatus() function.
Something like this:
`var (isValid, currentTask) = await GetStatus(sessionId);`

Reply:
changed the code as suggested